src/modules/gameapi.js

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+
+export async function getGameId(gameName) {
+  const { data } = await axios({
+    method: 'post',
+    url: 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/',
+    data: {
+      name: `${gameName}`,
+    },
+  });
+  const { result } = data;
+  const id = result.substring(14, result.indexOf(' added.'));
+  return id;
+}
+
+export async function postUser(id, data) {
+  const response = await axios({
+    method: 'post',
+    url: `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${id}/scores/`,
+    data,
+  });
+  return response;
+}
+
+export async function getUsers(id) {
+  const users = await axios({
+    method: 'get',
+    url: `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${id}/scores/`,
+  });
+  return users;
+}
```

Comment:
- [ ] It is required to use arrow functions for all your functions instead of function keyword. Kindly change this to arrow function and replicate it to every other concerned functions. Below is the section of the project requirements that stated this

![requirement-point](https://user-images.githubusercontent.com/26597432/195989320-0b30c93e-92c3-4946-940d-c11487ab523f.png)

- [ ] It is also required to use `fetch` method to send and receive data to and from API endpoints. Kindly use `fetch` method for this purpose instead of `axios`